packages/backend/tsconfig.json

```diff
@@ -5,7 +5,10 @@
     "outDir": "./dist",
     "rootDir": "./src"
   },
-  "include": ["src/**/*"],
+  "include": [
+    "src/**/*",
+    "migrations/20250410162417_add_interval_op_grant_table.js"
```

Comment:
why do we need this one in particular?

Reply:
We do not, good catch. It was a bad copy/paste.

Reply:
Fixed. Actually not a bad copy/paste. The filename was in my clipboard and I figured its was from that but it wasnt.

Apparently it ended up here from:

- move this migration out of migration file (I wanted to run without the migration without deleting it)
- move back in
- accept prompt to update imports for the migration file (which there are none)

this path gets added to the tsconfig.